notebooks/Tracking_drift_check.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<p>I would consider adding a Session to this plot to make it easier to navigate through the notebook using the Table of Contents.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>